comps/cores/proto/docarray.py

```diff
@@ -14,6 +14,21 @@ class TextDoc(BaseDoc):
     text: str
 
 
+class MultiTextDoc(BaseDoc):
```

Comment:
What about having this be `FactualityDoc`

Reply:
Changed.